packages/string-templates/src/helpers/javascript.ts

```diff
@@ -114,8 +120,49 @@ export function processJS(handlebars: string, context: any) {
       ),
     }
 
+    const logs: Log[] = []
+    // logging only supported on frontend
+    if (!isBackendService()) {
+      // this counts the lines in the wrapped JS *before* the user's code, so that we can minus it
+      const jsLineCount = frontendWrapJS(js).split(js)[0].split("\n").length
+      const buildLogResponse = (type: LogType) => {
+        return (...props: any[]) => {
+          if (!isTest()) {
```

Comment:
Why removing it for tests?

Reply:
Jest seems to detect the error/warn and end the test suite based on this - doesn't happen locally but seems to happen in CI